lunavl/sdk/errors/errors.py

```diff
@@ -134,11 +134,11 @@ def fromSDKError(cls, sdkError: FSDKErrorResult) -> "ErrorInfo":
 
         errorClassErrors = inspect.getmembers(cls, lambda err: isinstance(err, ErrorInfo))
 
-        def getError(sdkError: FSDKErrorResult):
-            attrs = dir(sdkError)
+        def getError(sdkError_: FSDKErrorResult):
+            attrs = dir(sdkError_)
```

Comment:
I think that FaceEngine  removed differents error. Now  FSDKErrorResult has an uniforf attributes for error